pkg/config/specification.go

```diff
@@ -8,32 +8,22 @@ import (
-	APIPort             int    `envconfig:"API_PORT" default:"8081"`
-	Debug               bool   `envconfig:"DEBUG" description:"Enable debug mode"`
-	LogLevel            string `envconfig:"LOG_LEVEL" default:"info" description:"Log level"`
-	GraceTimeOut        int64  `envconfig:"GRACE_TIMEOUT" description:"Duration to give active requests a chance to finish during hot-reload"`
-	MaxIdleConnsPerHost int    `envconfig:"MAX_IDLE_CONNS_PER_HOST" description:"If non-zero, controls the maximum idle (keep-alive) to keep per-host."`
-	InsecureSkipVerify  bool   `envconfig:"INSECURE_SKIP_VERIFY" description:"Disable SSL certificate verification"`
-	// The Storage DSN, this could be `memory` or `redis`
-	StorageDSN string `envconfig:"STORAGE_DSN" default:"memory://localhost"`
-
-	// Path of certificate when using TLS
-	CertPathTLS string `envconfig:"CERT_PATH"`
-
-	// Path of key when using TLS
-	KeyPathTLS string `envconfig:"KEY_PATH"`
-
-	// Flush interval for upgraded Proxy connections
-	BackendFlushInterval time.Duration `envconfig:"BACKEND_FLUSH_INTERVAL" default:"20ms"`
-
-	// Defines the time period of how often the idle connections maintained
-	// by the proxy are closed.
-	CloseIdleConnsPeriod time.Duration `envconfig:"CLOSE_IDLE_CONNS_PERIOD"`
-
-	Database    Database
-	Statsd      Statsd
-	Credentials Credentials
+	Port                 int           `envconfig:"PORT" default:"8080" description:"Default application port"`
+	APIPort              int           `envconfig:"API_PORT" default:"8081" description:"Admin API port"`
+	Debug                bool          `envconfig:"DEBUG" description:"Enable debug mode"`
+	LogLevel             string        `envconfig:"LOG_LEVEL" default:"info" description:"Log level"`
+	GraceTimeOut         int64         `envconfig:"GRACE_TIMEOUT" description:"Duration to give active requests a chance to finish during hot-reload"`
+	MaxIdleConnsPerHost  int           `envconfig:"MAX_IDLE_CONNS_PER_HOST" description:"If non-zero, controls the maximum idle (keep-alive) to keep per-host."`
+	InsecureSkipVerify   bool          `envconfig:"INSECURE_SKIP_VERIFY" description:"Disable SSL certificate verification"`
+	StorageDSN           string        `envconfig:"STORAGE_DSN" default:"memory://localhost" description:"The Storage DSN, this could be 'memory' or 'redis'"`
+	CertPathTLS          string        `envconfig:"CERT_PATH" description:"Path of certificate when using TLS"`
+	KeyPathTLS           string        `envconfig:"KEY_PATH" description:"Path of key when using TLS"`
+	BackendFlushInterval time.Duration `envconfig:"BACKEND_FLUSH_INTERVAL" default:"20ms" description:"Flush interval for upgraded Proxy connections"`
+	CloseIdleConnsPeriod time.Duration `envconfig:"CLOSE_IDLE_CONNS_PERIOD" description:"Defines the time period of how often the idle connections maintained by the proxy are closed."`
+	Database             Database
+	Statsd               Statsd
+	Credentials          Credentials
+	Tracing              Tracing
```

Comment:
We need some docs on this. I think adding this to `Misc` section as I did for Monitoring should be fine.